processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryConfig.java

```diff
@@ -259,20 +310,8 @@ public GroupByQueryConfig withOverrides(final GroupByQuery query)
         ((Number) query.getContextValue(CTX_KEY_MAX_ON_DISK_STORAGE, getMaxOnDiskStorage())).longValue(),
         getMaxOnDiskStorage()
     );
-    newConfig.maxSelectorDictionarySize = Math.min(
-        ((Number) query.getContextValue(
-            CTX_KEY_MAX_SELECTOR_DICTIONARY_SIZE,
-            getMaxSelectorDictionarySize()
-        )).longValue(),
-        getMaxSelectorDictionarySize()
-    );
-    newConfig.maxMergingDictionarySize = Math.min(
-        ((Number) query.getContextValue(
-            CTX_KEY_MAX_MERGING_DICTIONARY_SIZE,
-            getMaxMergingDictionarySize()
-        )).longValue(),
-        getMaxMergingDictionarySize()
-    );
```

Comment:
Since these two are removed from QueryContext, should we document it in the release notes?

Reply:
Yes, we should. I added a release notes label.